apps/libexec/check.py

```diff
@@ -80,10 +80,12 @@ def cmd_distribution(args):
 			print_perfdata = False
 
 	expired = get_expired(qh)
-	if list(expired)[0] == -1:
-		return 1
+	expired_list = list(expired)
+	if (len(expired_list) > 0):
+		if expired_list[0] == -1:
+			return 1
```

Comment:
As far as I could see `get_expired` always returns a list already. So this could be shorter:

```
	if expired and expired[0] == -1:
		return 1
```

Reply:
That seems like a good solution and it seems to work. Updating